contributions/catalog/webapps/templates/contribute/contribution_details.html

```diff
@@ -166,16 +166,104 @@ <h4>Date modified</h4>
         </div>
     {% endif %}
 
-    {% if reviewer %}
+    {% if editor %}
         <a class="button button3" role="button" href="{{ url_for('contribute.contribution_edit', contribution_id=post.id) }}">Edit Contribution</a>
     {% endif %}
-</div>
-
-
-<div class="action">
 
+    {% if reviewer %}
+        <a class="button button4" role="button" href="{{ url_for('contribute.contribution_review', contribution_id=post.id, review=True) }}">Review Contribution</a>
+    {% endif %}
 </div>
 
+{% if review %}
```

Comment:
When you click on View Capability or Talent when reviewing, the URL is rewritten incorrectly and returns no permissions error.  

![image](https://user-images.githubusercontent.com/4306604/153073899-a0dce759-1200-4f35-846b-42d34d76fdd9.png)

Please see the URL below:

![image](https://user-images.githubusercontent.com/4306604/153074055-d5ff31de-18ca-4f04-a4b1-4056df23d2b4.png)

I think here we need to direct them to the Capability or Talent detailed view where the reviewer should be able to see all the details. Please feel free to create a follow-up issue to get this fixed. Thanks.

Reply:
The button redirects for viewing capability and talent are fixed.

Reply:
redirect to contribution detail page after submission of the review

Reply:
I like the second change too. Thanks.